### ISSUE TITLE
In iOS asking to set Team ID

### ISSUE THREAD

**Issue by shantimohan** [microsoftgraph/msgraph-training-xamarin#19]
When run in iOS, the app is breaking at `PCA = builder.Build(); `statement, asking to set 'team id'. No clue where to set it. I even tried Automatic Provisioning which didn't help.

**Comment by jasonjoh** [comment]
Sorry you're having trouble! Is this happening at run time or compile? Can you provide the full error and stack trace?

**Comment by shantimohan** [comment]
Fortunately, it is running perfectly in both iOS Simulator and iPhone. It is running after I did the following:

1. Yesterday till I posted here, it was not working for iOS. That was on my laptop.
2. Today morning after seeing your reply, I moved the code to Desktop, ran it in iOS Simulator, it is worked fine.
3. Then I tried on my laptop and there was no issue.
4. Then I tried on my desktop connecting iPhone and it worked fine.

I don't know what caused the issue yesterday. Now I will proceed to implement to access OneDrive, which was my intention.

ThanQ...
